x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/notify_badge/index.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RulesListNotifyBadgeWithApi } from './notify_badge_with_api';
+export { RulesListNotifyBadge } from './notify_badge';
+export type { RulesListNotifyBadgePropsWithApi } from './types';
+export {
+  SNOOZE_SUCCESS_MESSAGE,
+  UNSNOOZE_SUCCESS_MESSAGE,
+  SNOOZE_FAILED_MESSAGE,
+} from './translations';
+
+// eslint-disable-next-line import/no-default-export
+export { RulesListNotifyBadgeWithApi as default };
```

Comment:
nitpick: should we have this default export in the component and re-export here? Just to be consistent with other shared components